README.md

```diff
@@ -143,6 +144,51 @@ outlined in the [Building images on non SLE distributions](#building-images-on-n
 section.
 
 
+### Building images on-demand SLE instances in the public cloud
+
+When building container images on SLE instance that where launched as so-called
+"on-demand" or "pay as you go" instances on a Public Cloud (as AWS, GCE or Azure)
+some additional steps have to be performed.
+
+For installing packages and updates the "on-demand" public cloud instance are
+connected to a public cloud specific update infrastructure which is based around
+some RMT servers operated by SUSE on the various Public Cloud Providers.
+
+To be able access this update infrastructure the instances need to perform some
+additional steps to locate the required service and authenticate with them.
+(Some more details on this are outline in a
+[Blog Series on suse.com starting here](https://suse.com/c/a-new-update-infrastructure-for-the-public-cloud/).
+
+In order to be able to build containers on this type of instances a new service
+was introduced, that service is called "containerbuild-regionsrv" and will be
+available in the public cloud images provided through the Marketplaces of the
+various Public Cloud Providers. So before building an image this service has
+to be started on the public cloud instance
+
+```
+systemctl start containerbuild-regionsrv
+```
+
+In order to have it started automatically (e.g. after reboot) please use:
+
+```
+systemctl enable  containerbuild-regionsrv
+```
+
+The zypper plugins provided by container-suseconnect will then connect to this
+service for getting authentication details and information about which update
+server to talk to. In order for that to work the container as to be build with
+host networking enabled. I.e. you need to call `docker build` with `--network host`:
+
+```
+docker build --network host <builddir>
+```
+
+As the update infrastructure in the Public Clouds is based upon RMT the same
```

Comment:
This is just my personal taste, feel free to ignore:

> The update infrastructure in the Public Clouds is based upon RMT hence the same

Reply:
Personally I'd go with something like: `Since the update infrastructure in the Public Clouds is based on RMT, the same` (notice the comma after RMT).

Reply:
Yeah, that sounds even better. Thanks.